reana-ui/src/components/Notification.js

```diff
@@ -0,0 +1,57 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React, { useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Container, Message, Transition } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+import { clearError } from "../actions";
+import { getError } from "../selectors";
+
+import styles from "./Notification.module.scss";
+
+export default function Notification({ icon, message }) {
+  const dispatch = useDispatch();
+  const error = useSelector(getError);
+  const timer = useRef(null);
+
+  const hide = () => dispatch(clearError);
+  const visible = message || error ? true : false;
+
+  if (visible) {
+    clearTimeout(timer.current);
+    timer.current = setTimeout(() => hide(), 16000);
+  }
+
+  return (
+    <Transition visible={visible} duration={300}>
+      <Container text className={styles.container}>
+        <Message
+          icon={icon}
+          header={message || error?.message}
```

Comment:
What do you think about changing `header` to be something shorter and generic such as "An error has occurred" (since it is thought as a title), and then inside `content` we could add `error.message` for details?